GitCommands/Git/GitModule.cs

```diff
@@ -1845,18 +1814,12 @@ public bool UnstageFiles(IReadOnlyList<GitItemStatus> files, out string allOutpu
             return !wereErrors;
         }
 
-        /// <summary>
-        /// Batch unstage files using <see cref="ExecutableExtensions.RunBatchCommand(IExecutable, ICollection{BatchArgumentItem}, Action{BatchProgressEventArgs}, Action{StreamWriter})"/>.
-        /// </summary>
-        /// <param name="selectedItems">Selected file items.</param>
-        /// <param name="action">Progress update callback.</param>
-        /// <returns><see langword="true" /> if changes should be rescanned; otherwise <see langword="false" />.</returns>.
-        public bool BatchUnstageFiles(IEnumerable<GitItemStatus> selectedItems, Action<BatchProgressEventArgs>? action = null)
+        public bool BatchUnstageFiles(IEnumerable<GitItemStatus> files, Action<BatchProgressEventArgs>? progressCallback = null)
```

Comment:
Why change to a generic name like files that can be anything?

Reply:
The API arguments in `GitModule` were aligned with the signatures of the contracts (i.e., the API in `IGitModule`). All `IGItModule` API will soon live in GitExtensions.Extensibility library and looking through the prism of the API consumers using that library "selected" means very little (it's a UI concept, which is an implementation detail).

Reply:
The signature should be aligned with the current name instead.

Reply:
I disagree. `BatchUnstageFiles` expects a list of files to be unstaged. What does "selected" mean in this context?

Reply:
This is fine